src/components/Login/Login.jsx

```diff
@@ -132,7 +138,7 @@ class Login extends Component {
 								</Link>
 							</Grid>
 							<Grid item>
-								<Link href="#" variant="body2">
+								<Link href="/register" variant="body2">
```

Comment:
Er der en grund til at bruge href frem for "to" som bruges andre steder?

Reply:
altså det virker så det er vel ligegyldigt?

Reply:
Det virker sgu ikke hvis man bruger to i det her tilfælde